DEVELOPMENT.md

```diff
@@ -0,0 +1,94 @@
+# KCP-Edge Contributor Instructions
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl (version range expected: 1.23-1.25)
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+After cloning this repo (see [instructions](#1-clone-the-kcp-edge-repo)), run the following script to install the required package (Linux or MacOS):
+
+```bash
+hack/install_req.sh
```

Comment:
(1) I do not see this file.
(2) Does it install Docker?  I suspect not.  If I am right then the user should not be told that this script will simply "install the required packages".